### PR TITLE
Map reconstructed using MapKit Map, old fashioned MapView is removed.

### PR DESCRIPTION
**Pull request by anwar-ulhaq**
Please check functionality of app by pulling this branch on your PC. It has all the feature so far we have.
@darjapo @TaiNguyen2407 @mohammra @giaongo Sorry i deleted  a lot of your code.

### CODE REVIEW

File: Voice Note/Views/MapView.swift [hunk 1]
```diff
@@ -1,72 +0,0 @@
-import SwiftUI
```
Comment (anwar-ulhaq) [inline Voice Note/Views/MapView.swift:1]: @Tai @darjapo Really sorry for this file. You guys have put a lot of time and effort in this file.

File: Voice Note/Voice_NoteApp.swift [hunk 2]
```diff
@@ -5,7 +5,7 @@ import SwiftUI
 struct Voice_NoteApp: App {
     @StateObject var speechRecognizer = SpeechRecognizer()
     @StateObject var voiceNoteViewModel = VoiceNoteViewModel()
-    @StateObject var mapViewModel = MapViewModel()
```
Comment (anwar-ulhaq) [inline Voice Note/Voice_NoteApp.swift:8]: 1st instance of MapView

File: Voice Note/Voice_NoteApp.swift [hunk 3]
```diff
@@ -14,7 +14,7 @@ struct Voice_NoteApp: App {
                 ContentView()
                         .environment(\.managedObjectContext, coreDataService.getManageObjectContext())
                         .environment(\.coreData, coreDataService)
-            }.environmentObject(speechRecognizer).environmentObject(voiceNoteViewModel).environmentObject(MapViewModel())
+            }.environmentObject(speechRecognizer).environmentObject(voiceNoteViewModel)
         }
```
Comment (anwar-ulhaq) [inline Voice Note/Voice_NoteApp.swift:18]: 2nd Instance of MapView

**Review by giaongo — APPROVED** [review]
Changes are read and approved

**Review by mohammra — APPROVED** [review]

**Pull request merged**